keps/sig-storage/20190408-volume-scheduling-limits.md

```diff
@@ -212,14 +212,11 @@ For brevity - "old predicates" refers to now deprecated cloudprovider specific p
 ##### When CSI driver for same underlying storage type is installed on the node.
 
 * For Azure, GCEPD, AWS and Cinder - in-tree volume plugins will report their limits via `Node` object same as before.
-* Old predicates will be modified to perform an additional check of `CSINode` at very beginning of their code. If they detect that `CSINode` object contains limit of same underlying volume type, they will return early(with success) and `MaxCSIVolumeCountPred` will be responsible for counting both CSI and in-tree volumes of same type.
-
```

Comment:
Removed because I think this paragraph doesn't belong here since the migration is disabled. The same instruction is in the section below.